sources/Application/Utils/HelpLegend.h

```diff
@@ -0,0 +1,98 @@
+#include <cstdio>
+#include <cstring>
+
+static inline const char (*getHelpLegend(const char *fx))[30] {
+  static char result[3][30];
+  strcpy(result[0], fx);
+  strcpy(result[2], "bb at speed aa");
+  if (strcmp(fx, "KIL") == 0) {
```

Comment:
I think you're jumping extra hoops for no reason on how you match each command to it's help string. You're using the buffer with the decoded command text, rather than use a switch statement and match on the FourCC definitions.
`
switch(command) {
  case I_CMD_KILL:
    ...
    break;
  case I_CMD_LPOF:
    ...
    break;
`